### PR TITLE
deepmedic recipe

### PR DESCRIPTION
**Pull request by ThomasWollmann**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/deepmedic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - theano
+    - nose
+    - nibabel
+    - pp
+    - six
+    - scipy
+    - numpy
+  run:
+    - theano
+    - nose
```
Comment (bgruening) [inline recipes/deepmedic/meta.yaml]: is nose really needed as run dependency

Comment (ThomasWollmann) [inline recipes/deepmedic/meta.yaml]: probably not...

**Comment by druvus** [comment]
Seems you were misisng the pp dependency. I have added it #3690 so I will restart the test once it is available.

**Comment by ThomasWollmann** [comment]
Thanks @druvus!

**Comment by bgruening** [comment]
mh, https://travis-ci.org/bioconda/bioconda-recipes/jobs/197942786#L746 Seems that a specific version of pp is needed. :(
You could either patch setup.py or create an older pp package.

**Comment by druvus** [comment]
@ThomasWollmann you need to patch setup.py to get rid of `dependency_links` part.

**Comment by ThomasWollmann** [comment]
@druvus is it enough to just remove it?

**Comment by druvus** [comment]
I think so

**Comment by druvus** [comment]
Thanks @ThomasWollmann

**Pull request merged**